### PR TITLE
Alerting UI Updates

### LINKED ISSUE #3934: Alerting UI changes
### Issue Type

Inaccurate, misleading, or out-of-date information, Missing content for an existing feature

### Links to Affected Docs

https://www.astronomer.io/docs/astro/alerts

### The Issue or Context

We have a new Alerting UI that is being rolled out to users

### Fix or Proposed Change

Update doc to align with the UI changes.

### Additional Notes

_No response_

### Required Reviewers

corey/Paola/vinh

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3934

### CODE REVIEW

File: astro/alerts.md [hunk 1]
```diff
@@ -244,23 +246,43 @@ In the Astro UI, you can enable alerts from the **Workspace Settings** page.
     </TabItem>
     </Tabs>
 
-6. Add DAGs or tasks that your alert applies to.
+7. Click **Create notification channel**.
+
+## Step 3: Create your alert in the Astro UI
+
+1. In the Astro UI, click **Alerts**, and then the **Alerts** Tab.
```
Comment (merobi-hub) [inline astro/alerts.md:253]:
```suggestion
1. In the Astro UI, click on **Alerts** and then the **Alerts** Tab.
```

File: astro/alerts.md [hunk 2]
```diff
@@ -244,23 +246,43 @@ In the Astro UI, you can enable alerts from the **Workspace Settings** page.
...
 
-    - **Absolute Time**: Click **+ DAG** and choose the Deployment and the DAG that you want the alert to assess. Then, select the **Days of Week** that the alert should observe, the **Verification Time** when it should look for a DAG success, and the **Lookback Period** for how long it should look back for a verification time.
+    - **Absolute Time**: Select the **Days of Week** that the alert should observe, the **Verification Time** when it should look for a DAG success, and the **Lookback Period** for how long it should look back for a verification time.
 
     For example, if an alert has a **Verification Time** of 3:00 PM and a **Lookback Period** of 60 minutes, it will trigger whenever the given DAG does not produce a successful DAG run from 2:00 to 3:00 PM. Astro applies the times you specify based on the time zone of your current web browser session, then translates them to UTC in your Airflow environment.
 
-7. (Optional) Repeat Step 6 for additional DAGs or tasks that you want to alert on.
+4. Choose the alert **Severity**, either **Info**, **Warning**, **Critical**, or **Error**.
+
+5. Define the conditions for when you want your alert to send a notification.
```
Comment (merobi-hub) [inline astro/alerts.md:271]:
```suggestion
5. Define the conditions on which you want your alert to send a notification.
```

File: astro/alerts.md [hunk 3]
```diff
@@ -244,23 +246,43 @@ In the Astro UI, you can enable alerts from the **Workspace Settings** page.
...
+
+  - Select the **Attribute**, **Operator**, and **DAGs** to define when you want Astro to send an Alert and which DAGs that you want the alert to apply to.
+
+## Step 3: (Optional) Change an alert name
+
+After you select a DAG that you want to apply an alert to, Astro automatically generates a name for your alert. However, you can choose to change the name for your alert.
```
Comment (merobi-hub) [inline astro/alerts.md:277]:
```suggestion
After you select a DAG that you want to apply an alert to, Astro automatically generates a name for your alert. However, you can choose to change the name of your alert.
```

**Review by merobi-hub — APPROVED** [review]
Very nit-picky punctuation changes you might want to make but LGTM

**Comment by lzdanski** [comment]
Confirmed that these changes can be published. Merging.

**Pull request merged**